src/components/resource/OcResource.vue

```diff
@@ -5,18 +5,10 @@
       :key="preview"
       :src="preview"
       class="oc-resource-preview"
-      aria-hidden="true"
       width="40"
       height="40"
     />
-    <oc-icon
-      v-else
-      key="resource-icon"
-      :name="resource.icon"
-      variation="system"
-      size="large"
-      aria-hidden="true"
-    />
+    <oc-icon v-else key="resource-icon" :name="resource.icon" variation="system" size="large" />
```

Comment:
Needs a rebase now. The `system` variation doesn't exist anymore. Use `passive` instead, which is also the new default (so could could also just omit the variation attribute). See https://github.com/owncloud/owncloud-design-system/blob/917a6c8c54191c80c35d6aabb9314b03b61218f8/src/components/OcIcon.vue#L94